cmd/snap-update-ns/change.go

```diff
@@ -601,24 +611,36 @@ func neededChanges(currentProfile, desiredProfile *osutil.MountProfile) []*Chang
 	// 3. Perform all the remaining desired mounts
 
 	var newDesiredEntries []osutil.MountEntry
+	var newIndependentDesiredEntries []osutil.MountEntry
 	// Indexed by mount point path.
 	addedDesiredEntries := make(map[string]bool)
 	// This function is idempotent, it won't add the same entry twice
 	addDesiredEntry := func(entry osutil.MountEntry) {
 		if !addedDesiredEntries[entry.Dir] {
+			logger.Debugf("adding entry: %s", entry)
 			newDesiredEntries = append(newDesiredEntries, entry)
 			addedDesiredEntries[entry.Dir] = true
 		}
 	}
+	addIndependentDesiredEntry := func(entry osutil.MountEntry) {
+		if !addedDesiredEntries[entry.Dir] {
+			logger.Debugf("adding independent entry: %s", entry)
```

Comment:
nitpick^2: if you also printed `entry.XSnapdOrigin()` in this line, you could get rid of the line
```
logger.Debugf("- overname entry: %v", entry)
```
below.

Reply:
Actually, this is printed already, as it's part of the mount options; so you could just remove the line below, I don't think it adds anything.